src/lib/containers/oauth/CreateOAuthClient.tsx

```diff
@@ -121,15 +133,15 @@ export const CreateOAuthModal: React.FunctionComponent<
         setUpdatedClient(oAuthClient)
         if (warnTrigger === true) {
           setIsShowingConfirmModal(true)
-          hide()
         } else {
           if (isEdit) {
-            mutate({ action: 'UPDATE', client: oAuthClient })
+            updateClient(oAuthClient)
           } else {
-            mutate({ action: 'CREATE', client: oAuthClient })
+            createClient(oAuthClient)
```

Comment:
Should probably await these mutation calls so the modal doesn't hide right away.